Unable to compile examples

I am not able to compile `orbclient`. I just keep getting errors saying that four features cannot be used on the stable release channel.

I'm running [Pop!_OS](https://system76.com/pop), a variant of Ubuntu, and I've installed Rust 1.24.1 using the script [over on rust-lang.org](https://www.rust-lang.org/en-US/install.html) and installed SDL2 via `apt install libsdl2-dev` per [the instructions](https://github.com/Rust-SDL2/rust-sdl2#linux).

What did I miss?

Here's my full output:
```
λ cargo run --example widgets               
    Updating registry `https://github.com/rust-lang/crates.io-index`
 Downloading orbclient v0.3.13                                                  
 Downloading lazy_static v0.2.11                                                
 Downloading orbfont v0.1.8                                                     
 Downloading cssparser v0.16.1                                                  
 Downloading orbimage v0.1.16                                                   
 Downloading rayon v0.8.2                                                       
 Downloading time v0.1.39                                                       
 Downloading rayon-core v1.4.0                                                  
 Downloading crossbeam-deque v0.2.0                                             
 Downloading num_cpus v1.8.0                                                    
 Downloading rand v0.4.2                                                        
 Downloading libc v0.2.39                                                       
 Downloading lazy_static v1.0.0                                                 
 Downloading crossbeam-utils v0.2.2                                             
 Downloading crossbeam-epoch v0.3.0                                             
 Downloading cfg-if v0.1.2                                                      
 Downloading arrayvec v0.4.7                                                    
 Downloading scopeguard v0.3.3                                                  
 Downloading nodrop v0.1.12                                                     
 Downloading memoffset v0.2.1                                                   
 Downloading rusttype v0.2.3                                                    
 Downloading linked-hash-map v0.5.1                                             
 Downloading stb_truetype v0.2.2                                                
 Downloading byteorder v1.2.1                                                   
 Downloading matches v0.1.6                                                     
 Downloading cssparser-macros v0.3.2                                            
 Downloading phf v0.7.21                                                        
 Downloading procedural-masquerade v0.1.5                                       
 Downloading syn v0.12.14                                                       
 Downloading quote v0.4.2                                                       
 Downloading proc-macro2 v0.2.3                                                 
 Downloading phf_codegen v0.7.21                                                
 Downloading unicode-xid v0.1.0                                                 
 Downloading phf_shared v0.7.21                                                 
 Downloading phf_generator v0.7.21                                              
 Downloading siphasher v0.2.2                                                   
 Downloading rand v0.3.22                                                       
 Downloading syn v0.11.11                                                       
 Downloading quote v0.3.15                                                      
 Downloading synom v0.11.3                                                      
 Downloading unicode-xid v0.0.4                                                 
 Downloading resize v0.2.0                                                      
 Downloading image v0.12.4                                                      
 Downloading jpeg-decoder v0.1.14                                               
 Downloading scoped_threadpool v0.1.9                                           
 Downloading num-rational v0.1.42                                               
 Downloading num-iter v0.1.35                                                   
 Downloading gif v0.9.2                                                         
 Downloading enum_primitive v0.1.1                                              
 Downloading num-traits v0.1.43                                                 
 Downloading png v0.6.2                                                         
 Downloading rayon v1.0.0                                                       
 Downloading either v1.4.0                                                      
 Downloading num-traits v0.2.1                                                  
 Downloading num-integer v0.1.36                                                
 Downloading color_quant v1.0.0                                                 
 Downloading lzw v0.10.0                                                        
 Downloading deflate v0.7.17                                                    
 Downloading inflate v0.1.1                                                     
 Downloading bitflags v0.7.0                                                    
 Downloading adler32 v1.0.2                                                     
 Downloading sdl2 v0.31.0                                                       
 Downloading num v0.1.42                                                        
 Downloading sdl2-sys v0.31.0                                                   
   Compiling rayon-core v1.4.0                                                  
   Compiling procedural-masquerade v0.1.5
   Compiling libc v0.2.39
   Compiling quote v0.3.15
   Compiling inflate v0.1.1
   Compiling lazy_static v0.2.11
   Compiling resize v0.2.0
   Compiling bitflags v0.7.0
   Compiling color_quant v1.0.0
   Compiling siphasher v0.2.2
   Compiling matches v0.1.6
   Compiling num-traits v0.2.1
   Compiling scopeguard v0.3.3
   Compiling scoped_threadpool v0.1.9
   Compiling lazy_static v1.0.0
   Compiling byteorder v1.2.1
   Compiling linked-hash-map v0.5.1
   Compiling adler32 v1.0.2
   Compiling cfg-if v0.1.2
   Compiling either v1.4.0
   Compiling lzw v0.10.0
   Compiling memoffset v0.2.1
   Compiling nodrop v0.1.12
   Compiling unicode-xid v0.0.4
   Compiling unicode-xid v0.1.0
   Compiling phf_shared v0.7.21
   Compiling rand v0.4.2
   Compiling num_cpus v1.8.0
   Compiling time v0.1.39
   Compiling sdl2-sys v0.31.0
   Compiling crossbeam-utils v0.2.2
   Compiling deflate v0.7.17
   Compiling stb_truetype v0.2.2
   Compiling arrayvec v0.4.7
   Compiling proc-macro2 v0.2.3
   Compiling synom v0.11.3
   Compiling phf v0.7.21
   Compiling gif v0.9.2
   Compiling num-integer v0.1.36
   Compiling num-traits v0.1.43
   Compiling syn v0.11.11
   Compiling crossbeam-epoch v0.3.0
   Compiling rusttype v0.2.3
   Compiling rand v0.3.22
   Compiling enum_primitive v0.1.1
   Compiling quote v0.4.2
   Compiling num-iter v0.1.35
   Compiling num-rational v0.1.42
   Compiling phf_generator v0.7.21
   Compiling num v0.1.42
   Compiling phf_codegen v0.7.21
   Compiling crossbeam-deque v0.2.0
   Compiling syn v0.12.14
   Compiling png v0.6.2
   Compiling sdl2 v0.31.0
   Compiling rayon v0.8.2
   Compiling rayon v1.0.0
   Compiling cssparser v0.16.1
   Compiling orbclient v0.3.13
error[E0554]: #![feature] may not be used on the stable release channel
 --> /home/keawade/.cargo/registry/src/github.com-1ecc6299db9ec823/orbclient-0.3.13/src/lib.rs:3:1
  |
3 | #![feature(alloc)]
  | ^^^^^^^^^^^^^^^^^^

error[E0554]: #![feature] may not be used on the stable release channel
 --> /home/keawade/.cargo/registry/src/github.com-1ecc6299db9ec823/orbclient-0.3.13/src/lib.rs:4:1
  |
4 | #![feature(asm)]
  | ^^^^^^^^^^^^^^^^

error[E0554]: #![feature] may not be used on the stable release channel
 --> /home/keawade/.cargo/registry/src/github.com-1ecc6299db9ec823/orbclient-0.3.13/src/lib.rs:5:1
  |
5 | #![feature(const_fn)]
  | ^^^^^^^^^^^^^^^^^^^^^

error[E0554]: #![feature] may not be used on the stable release channel
 --> /home/keawade/.cargo/registry/src/github.com-1ecc6299db9ec823/orbclient-0.3.13/src/lib.rs:6:67
  |
6 | #![cfg_attr(all(not(feature="no_std"), not(target_os = "redox")), feature(const_ptr_null_mut))]
  |                                                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to 4 previous errors

error: Could not compile `orbclient`.
warning: build failed, waiting for other jobs to finish...
error: build failed
```

--- a/README.md
+++ b/README.md
@@ -30,3 +30,4 @@ You can start the adventure example by executing the following command:
 ```text
 cargo run --example widgets
 ```
+

--- a/README.md
+++ b/README.md
@@ -19,6 +19,8 @@ However you also need to have the SDL2 libraries installed on your
 system.  The best way to do this is documented [by the SDL2
 crate](https://github.com/AngryLawyer/rust-sdl2#user-content-requirements).
 
+You will also need to use the [nightly build of Rust](https://github.com/rust-lang-nursery/rustup.rs/blob/master/README.md#working-with-nightly-rust) to use orbtk. 
+
 ## Examples
 
 You find the examples in the `examples/` directory.

